src/apis/Animated.re

```diff
@@ -36,8 +36,7 @@ module ValueAnimations = (Val: Value) => {
         ~iterations: int=?,
         unit
       ) =>
-      config =
-      "";
+      config;
```

Comment:
Please mirror this change (and others below) in the .md file.

Reply:
✅